frontend/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "frontend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack-dev-server",
+    "watch": "webpack --watch"
```

webpack 명령어는 프로덕션으로 빌드시킬 때 사용되는 용도여서 실시간으로 봐야 할 때는 dev server를 사용하고 watch 옵션은 제거하여도 됩니다!

```suggestion
    "build": "webpack"
```